### DISCUSSION TITLE
Relicense the specification

### DISCUSSION THREAD

**Discussion by jotaen** [jotaen/klog#359] [Something else]
In the [Specification it currently says](https://github.com/jotaen/klog/blob/64ac115f52e4bf3a9bc40491f4687fc9c03d21ce/Specification.md?plain=1#L7):

> It is free and open-source software distributed under the MIT-License.

The license of the CLI tool is MIT, and back in the day, I just took that over to the specification document without much thinking.

I’m actually not sure anymore, though, whether the MIT license makes sense for such a document. The spec is technically not software, after all, but it’s a document describing a “standard” (that of klog, namely).

Mozilla for example uses a combination of [CC0](https://creativecommons.org/choose/zero/) and [OWFa](https://www.openwebfoundation.org/the-agreements/the-owf-1-0-agreements-granted-claims/owfa-1-0) for their standards documents:
- CC0 releases the document into the public domain, which allows maximum freedom.
- OWFa declares that the document is patent-free, so it can be safely used by everyone.

They have [some background info here](https://wiki.mozilla.org/Standards/license).

The license header of the file format spec could be this then:

> Per [CC0](http://creativecommons.org/publicdomain/zero/1.0/), to the extent possible under law, the editors have waived all copyright and related or neighbouring rights to this work. In addition, as of March 2022, the editors have made this specification available under the [Open Web Foundation Agreement (OWFa) Version 1.0](https://www.openwebfoundation.org/the-agreements/the-owf-1-0-agreements-granted-claims/owfa-1-0).

Before making any change, I need to read up more about this topic. Also, @vladdeSV since you contributed to the spec, I’d need your consent for a license change.

**Comment by vladdeSV** [comment]
You have my permission to change the license 👍

**Comment by jotaen** [comment]
Scheduled for the next release https://github.com/jotaen/klog/pull/170.
